### ISSUE TITLE
Build fails with Java > 8

### ISSUE THREAD

**Issue by JulianKniephoff** [opencast/annotation-tool#473]
With Java 11 I see for example

```
[ERROR] Failed to execute goal de.empulse.eclipselink:staticweave-maven-plugin:1.0.0:weave (default) on project annotation-tool-impl: Execution default of goal de.empulse.eclipselink:staticweave-maven-plugin:1.0.0:weave failed: An API incompatibility was encountered while executing de.empulse.eclipselink:staticweave-maven-plugin:1.0.0:weave: java.lang.ExceptionInInitializerError: null
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>de.empulse.eclipselink:staticweave-maven-plugin:1.0.0
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/jules/.m2/repository/de/empulse/eclipselink/staticweave-maven-plugin/1.0.0/staticweave-maven-plugin-1.0.0.jar
[ERROR] urls[1] = file:/home/jules/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.jpa/2.6.4/org.eclipse.persistence.jpa-2.6.4.jar
[ERROR] urls[2] = file:/home/jules/.m2/repository/org/eclipse/persistence/javax.persistence/2.1.1/javax.persistence-2.1.1.jar
[ERROR] urls[3] = file:/home/jules/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.asm/2.6.4/org.eclipse.persistence.asm-2.6.4.jar
[ERROR] urls[4] = file:/home/jules/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.antlr/2.6.4/org.eclipse.persistence.antlr-2.6.4.jar
[ERROR] urls[5] = file:/home/jules/.m2/repository/org/glassfish/javax.json/1.0.4/javax.json-1.0.4.jar
[ERROR] urls[6] = file:/home/jules/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.jpa.jpql/2.6.4/org.eclipse.persistence.jpa.jpql-2.6.4.jar
[ERROR] urls[7] = file:/home/jules/.m2/repository/org/eclipse/persistence/org.eclipse.persistence.core/2.6.4/org.eclipse.persistence.core-2.6.4.jar
[ERROR] urls[8] = file:/home/jules/.m2/repository/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[project>org.opencast.annotate:annotation-tool-api:0-SNAPSHOT, parent: ClassRealm[maven.api, parent: null]]]
[ERROR] 
[ERROR] -----------------------------------------------------
[ERROR] : NullPointerException
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :annotation-tool-impl
```

**Comment by JulianKniephoff** [comment]
I made CI use Java 8 for now: bd47f08545f8e3ce29642dbf6083ae9c0d5f0f0b

But in the long run, we need to fix this.

**Comment by JulianKniephoff** [comment]
Should be fixed in 5f674fbd8d3190bcdd536ebb95d41570619cca99
